Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 
 Bugfixes:
  * Code Generator: Defensively pad memory for ``type(Contract).name`` to multiples of 32.
+ * Function calls with named arguments now work with overloaded functions
```

Comment:
Compiler feature?

Reply:
I first thought it would be a bug fix as the issue was labeled as bug.

Reply:
Yeah, you can probably argue.